tests/test_wave_05.py

```diff
@@ -52,13 +52,18 @@ def test_new_genre_rec_from_empty_friends():
             }
         ]
     }
+     # Act
+    recommendations = get_new_rec_by_genre(sonyas_data)
+
+    assert len(recommendations) == 0
```

Comment:
What if we did assert `len(genre_recommendations) == []`? This more readily declares that the genre_recommendations should be an empty list. Also for testing, if someone returned, say an empty string, this test would technically pass.